Runner/JobManager.py

```diff
@@ -0,0 +1,334 @@
+from azure.storage.blob.models import BlobBlock, ContainerPermissions, ContentSettings
+from pathlib import Path
+import azure.batch.models as batchmodels
+import traceback
+import asyncio
+import Utils
+import os
+import datetime
+import functools
+import datetime
+import time
+import CustomTemplateFactory as ctm
+"""
+This module is responsible for creating, submitting and monitoring the pools and jobs
+
+"""
+
+
+_time = str(datetime.datetime.now().day) + "-" + \
+    str(datetime.datetime.now().hour) + "-" + \
+    str(datetime.datetime.now().minute)
+
+
+async def submit_job(batch_service_client, template, parameters):
+    """
+    Submits a Job against the batch service.
+
+    :param batch_client: The batch client to use.
+    :type batch_client: `batchserviceclient.BatchServiceClient`
+    :param str template: the json desciption of the job
+    :param str the parameters of the job
+    """
+    try:
+        loop = asyncio.get_event_loop()
+        job_json = batch_service_client.job.expand_template(
+            template, parameters)
+        jobparameters = batch_service_client.job.jobparameter_from_json(
+            job_json)
+        job = await loop.run_in_executor(None, functools.partial(batch_service_client.job.add, jobparameters))
+    except batchmodels.batch_error.BatchErrorException as err:
+        print(
```

Comment:
try to use `logging` instead of print

Reply:
Yep switching every print statement over to use logging